Tests/SentryTests/SentryLegacyProfiler+Test.h

```diff
@@ -0,0 +1,35 @@
+#import "SentryProfilingConditionals.h"
+
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+
+#    import "SentryLegacyProfiler.h"
+#    import <Foundation/Foundation.h>
+
+@class SentryProfiler;
+
+@interface
+SentryLegacyProfiler ()
+
+#    if defined(TEST) || defined(TESTCI)
+
++ (SentryProfiler *_Nullable)getCurrentProfiler;
```

Comment:
This wasn't previously `_Nullable` but it should've been, since the underlying global var is nullable. This affecting just a couple callsites in test code, some of which now use `XCTUnwrap`.